scripts/select_appointments.sql

```diff
@@ -1 +1,16 @@
 -- retrieve appointment information
+SELECT a.time AS time, d.family_name
```

Comment:
This and the next `SELECT` statement will cause syntax errors because `AS` needs to follow a table name.

```sql
SELECT p.family_name, p.given_name, a.time
FROM appointments AS a
INNER JOIN patients AS p
  ON a.patient_id = p.id
INNER JOIN doctors AS d
  ON a.doctor_id = d.id
WHERE d.family_name = 'Hubbard' AND d.given_name = 'Spencer';

SELECT d.family_name, d.given_name, a.time
FROM appointments AS a
INNER JOIN doctors AS d
  ON a.doctor_id = d.id
INNER JOIN patients p
  ON a.patient_id = p.id
WHERE p.family_name = 'Wilcox' AND p.given_name = 'Marsha';
```